docs/docs/50-running/02-networking.md

```diff
@@ -120,3 +120,49 @@ To highlight the level of control this gives you, consider the following:
 If you set your cluster domain to `my-company.com`, deployed into the default `acorn` namespace, named your app `blog`, and the container name in the Acornfile was `default`, the published endpoint for your app would be: `http://blog.my-company.com`
 
 In addition to this method of controlling your endpoint, you can publish to an explicit external name using the `--publish` flag. See the [publishing ports](#publish-individual-ports) section for more details.
+## Routers
+If you have two containers normally they would be exposed as two different HTTP services. The router feature allows you to expose these two containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this below.
+
+Starting with a sample Acornfile that exposes two services
+```acorn
```

Comment:
Follow up from the previous nit-pick, could we try to make the formatting with newlines match the rest of the document? Applies to the rest of this document.

```suggestion

```acorn

```

Reply:
fixed